extras/gen_release_candidate_changes.py

```diff
@@ -0,0 +1,64 @@
+#!/bin/env python
```

Comment:
Add some docs on how to execute and give an example of output.

Reply:
Done.